rgw/standalone/s3cmd-setup-v3-auto-dfg.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python3
+'''
+> download this script for auto configure s3cmd and create a bucket and upload 3 types of files.
+> use - chmod +x script_name.py and then run - ./script_name.py
+> compatible with python version 3
+> script functionalities
+        - enable epel
+        - install python2-pip, upgarde pip, install s3cmd
+        - create a user = 'operator'
+        - configure .s3cfg file
+        - create 3 files(500kb, 500mb, 4gb) and upload that in bucket (with 50000 objects)
+        - delete 3 files created in the system
+'''
+import os, sys, random
+print('enable the epel repo or by default it may be installed')
+enable_epel = os.system('yum-config-manager --enable epel')
+print(enable_epel)
+print("----------------------------------------------------------------------")
+install_python3_pip = os.system('yum install python3-pip -y')
+print(install_python3_pip)
+print("----------------------------------------------------------------------")
+upgrade_pip = os.system('pip3 install --upgrade pip')
+print(upgrade_pip)
+print("----------------------------------------------------------------------")
+install_s3cmd = os.system('yum install s3cmd -y; pip3 install s3cmd')
+print(install_s3cmd)
+print("----------------------------------------------------------------------")
+access_key='12345'
+secret_key='67890'
+create_user = os.system("radosgw-admin user create --uid=\"operator\" --display-name=\"S3 Operator\" --email=\"operator@example.com\" --access_key={} --secret={}".format(access_key, secret_key))
+print(create_user)
+print("----------------------------------------------------------------------")
+print('It will remove the default /root/.s3cfg file')
+s3cmd_configure = os.system('s3cmd --configure --dump-config > /root/.s3cfg')
+print(s3cmd_configure)
+print("----------------------------------------------------------------------")
+print('Make some changes to .s3cfg file')
+hostname = os.uname()[1]
+port_number='8080'
+os.system('sed -i -e \'s,^host_base *=.*,host_base = http://{}:{},;s,host_bucket *=.*,host_bucket = http://{}:{},;s,website_endpoint *=.*,website_endpoint = http://%(bucket)s.{}-%(location)s,;s,access_key *=.*,access_key = {},;s,secret_key *=.*,secret_key = {},;s,use_https *=.*,use_https = False,;s,gpg_command *=.*,gpg_command = /usr/bin/gpg,;s,progress_meter *=.*,progress_meter = True,;s,proxy_port *=.*,proxy_port = 0,\' /root/.s3cfg'.format(hostname, port_number, hostname, port_number, hostname, access_key, secret_key))
+s3cmd_work = os.system('s3cmd ls')
```

Comment:
I would suggest to use subprocess to get system commands execution and also retrieve exit status